test/src/test/java/org/corfudb/recovery/FastSmrMapsLoaderTest.java

```diff
@@ -0,0 +1,243 @@
+package org.corfudb.recovery;
+
+import com.google.common.reflect.TypeToken;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.LogUnitClient;
+import org.corfudb.runtime.clients.SequencerClient;
+import org.corfudb.runtime.collections.SMRMap;
+import org.corfudb.runtime.object.CorfuCompileProxy;
+import org.corfudb.runtime.object.ICorfuSMR;
+import org.corfudb.runtime.object.VersionLockedObject;
+import org.corfudb.runtime.object.transactions.TransactionType;
+import org.corfudb.runtime.view.AbstractViewTest;
+import org.corfudb.runtime.view.ObjectsView;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by rmichoud on 6/16/17.
+ */
+public class FastSmrMapsLoaderTest extends AbstractViewTest {
+
```

Comment:
I think the tests should test cases where you process more than a multiple of the batch size